Added 12 new Q&A pairs to the Support knowledgebase section (#1081)
wandb/docs: content/support/is_there_any_way_to_overwrite_the_logs_from_previous_steps.md

---

> title: Is there any way to overwrite the logs from previous steps?

Reviewer: ```suggestion
title: How can I overwrite the logs from previous steps?
```

toc_hide: true
type: docs
tags:
  - logs
  - metrics
---

> Yes, you can use our features [forking]({{< relref "guides/models/track/runs/forking.md" >}}) and [rewind]({{< relref "guides/models/track/runs/rewind.md" >}}).

Reviewer: ```suggestion
To overwrite logs from previous steps, use [forking]({{< relref "guides/models/track/runs/forking.md" >}}) and [rewind]({{< relref "guides/models/track/runs/rewind.md" >}}).
```
